python/pyspark/errors/exceptions/base.py

```diff
@@ -137,11 +137,11 @@ def _log_exception(self) -> None:
             if query_context.contextType().name == "DataFrame":
                 logger = PySparkLogger.getLogger("DataFrameQueryContextLogger")
                 call_site = query_context.callSite().split(":")
-                line_no = call_site[1] if len(call_site) == 2 else ""
+                line_number = call_site[1] if len(call_site) == 2 else ""
```

Comment:
What about `lineno` since this is Python specific

Reply:
or `line` to be consistent with `SQLQueryContext`?

Reply:
`line` sounds good to me. Just applied the comment. Thanks!